code-challenges.test.js

```diff
@@ -14,33 +17,148 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const hitchhikersCharacters = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
+// const hitchhikersCharacters = [
+//   { name: "ford prefect", occupation: "a hitchhiker" },
+//   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+//   { name: "arthur dent", occupation: "a radio employee" }
+// ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+// describe("peopleList", () => {
+//     it("creates a function that takes an array of objects and returns an array with a sentence about each person with their name capitilized", () => {
+//     expect(peopleList(hitchhikersCharacters)).toEqual("Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee.")
+//  })
+// })
+
+//Red test- Good Failure
+// FAIL  ./code-challenges.test.js
+// peopleList
+//   ✕ creates a function that takes an array of objects and returns an array with a sentence about each person with their name capitilized (1 ms)
+
+// ● peopleList › creates a function that takes an array of objects and returns an array with a sentence about each person with their name capitilized
+
+//   ReferenceError: peopleList is not defined
+
+
 // b) Create the function that makes the test pass.
+// PseudoCode
+// Input: hitchhikerCharacters
+// Output : ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+// Process: create a function called peopleList and pass in a parameter of an array. 
+// Create another variable called newList that calls in a new function that will store the array that has been iterated through using .map ? I tried several ways to figure out how to get the array.map to store to a function, the internet search suggested to use the arrow syntax
+// in so that we can return a terate through the function using a .map method
+//create a variable that will store a new array use a method to take each key value pair and return them as a sentence ? using the .split method
+//then add the .map method again to capitilize the word at the 0 index
+//use a method to capitilize a the first word in the sentence in the array
+//use the method .join to turn into array once more
+//return 
+
+
+
+// const peopleList = (array) => {
+//   let newList => { 
```

Comment:
I think your research confused you even more, especially the part about calling a new function in a variable to store an array. You have the knowledge and tools you need for this challenge, so don't overthink it. I suggest you break this down into small pieces, and figure out what needs to happen for each step, and work through it step by step. For example:
1. I have an array filled with objects, I want to access each object, so I'll iterate using .map
2. What do I want to do for each object? I want to access the name key, so I'll use dot notation
3. What do I want to do for each name? I need to split the string into an array of 2 separate words, so I'll use .split(" ") with the space in between the quotes, so that it will be strings of each word instead of each letter
4. Now that it's an array of 2 words, I need to iterate again to access each word, so I'll use .map
5. What to do for each word? I want to capitalize the first letter, I can do that by using .charAt(0), then I need to add the rest of the word to the end of the capitalized letter, so I'll do + word.slice(1).

So on and so forth.